book/website/collaboration/research-infrastructure-roles/rse.md

```diff
@@ -1,25 +1,44 @@
 (cl-infrastructure-rse)=
 # Research Software Engineer: Overview
 
-Research Software Engineers (RSEs) are programmers with scientific backgrounds who play increasingly critical roles in the conduct of research and production of research software tools. 
-They are highly skilled team members who can also conduct their own research as part of their role. 
-
+Research Software Engineers (RSEs) are a broad group of people who combine software engineering expertise with an intimate understanding of research to contribute to the academic community.
+RSEs are involved in developing, maintaining, and deploying research software so that it is widely useable, well-documented, and produces reproducible outputs.
+While RSEs are increasingly critical to academic research, they often come from a variety of backgrounds and make diverse contributions which may not fit neatly into current formal academic roles, creating barriers to recognition and advancement.
+Therefore, this short guide aims to raise awareness of the RSE role, outline pathways for personal development and institutional recognition for RSEs, and list active community groups promoting RSE awareness.
+<!-- 
+To footnote:
+This overview combines parts of the following descriptions and definitions of the RSE role:
+https://rse-aunz.github.io
+https://nl-rse.org/posts/2017-06-13-what-is-rse
+https://us-rse.org/about/what-is-an-rse/
+https://society-rse.org/about/
+-->
 
 (cl-infrastructure-rse-role)=
 ## What do Research Software Engineers do? 
-RSEs work on research projects. 
-They may be assigned to projects based on skills or background from a pool of other RSEs, or be specifically hired to work on a project. 
+As already mentioned, RSEs can come from a wide range of backgrounds.
+It is important to mention, however, that the overarching characteristic element of all RSEs is that the output of their research is not papers, but software and toolboxes (which might be accompanied by an overview paper or published manual).
+Thus, an important step to allow the acknowledgement of research software work  has been setting up policies that allow the assignment of DOIs for dataset and software for example by the FFORCE11 Software Citation Working Group  {cite:ps}`smith2016software,Cohen2021FourPillars`.
```

Comment:
hmmm I don't think this is the right sub-heading for this point on DOIs - you might want to move later to the challenges section (also check if we have chapters on that already too!)